Sources/ShapedArray/Operations/pack.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  File.swift
+//  
+//
+//  Created by Jaap Wijnen on 10/08/2022.
+//
+
+extension ShapedArray {
+    static func pack(_ values: [ShapedArray], axis: Int = 0) -> ShapedArray {
```

Comment:
Do we really wish to have every operator live in it's own file? This seems like overkill and makes it harder to navigate through files.

Personally, if I want to know the set of operators I look at Docs not the file names, and if I want to look at implementation I want to see that category of things (in this case Operations/Operators) all at once, and I can use cmd+f to find something specific.

Reply:
Another way to state this just to try to be clear, if one wishes to get a lay of the land of operator implementations would they prefer clicking file by file (or in terminal worse) or just scrolling? Scrolling is a much easier movement and visual context is more easily maintained than moving between files.

Reply:
Please don’t put one operator per file; that’s not what files are supposed to be. Unless you have a super complex algorithm with many parts that’s distinct from anything else.

Reply:
Try looking at what S4TF does as a starting point, then seeing what solution works best for us.

Reply:
Let's discuss this further in https://github.com/swift-numerics-workgroup/swift-shaped-array/pull/3#discussion_r945154932.